infra-templates/network-diagnostics/1/rancher-compose.yml

```diff
@@ -1,5 +1,19 @@
 .catalog:
-  version: 0.0.1
+  version: 0.0.2
+  minimum_rancher_version: v1.6.5-rc1
+  services:
+  network-diagnostics:
```

Comment:
Why is healthcheck in the docker-compose and rancher-compose.yml? Pick 1 place. Also, if it's in rancher-compose.yml, you need to place it outside of the catalog section.